cherry_picker/cherry_picker/cherry_picker.py

```diff
@@ -155,13 +155,19 @@ def push_to_remote(self, base_branch, head_branch, commit_message=""):
         except subprocess.CalledProcessError:
             click.echo(f"Failed to push to {self.pr_remote} \u2639")
         else:
-            if os.getenv("GH_AUTH"):
-                self.create_gh_pr(base_branch, head_branch, commit_message)
+            gh_auth = os.getenv("GH_AUTH")
+            if gh_auth:
+                self.create_gh_pr(base_branch, head_branch, commit_message,
+                                  gh_auth)
             else:
                 self.open_pr(self.get_pr_url(base_branch, head_branch))
 
-    def create_gh_pr(self, base_branch, head_branch, commit_message):
-        request_headers = create_headers(self.username, oauth_token=os.getenv("GH_AUTH"))
+    def create_gh_pr(self, base_branch, head_branch, commit_message, gh_auth):
```

Comment:
Is there any worry the order of these arguments will be non-obvious? I.e. should any of them be keyword-only arguments?

Reply:
Hmm, ok I made commit_message and gh_auth as keyword only arguments.